### ISSUE TITLE
Installation on OSX fails

### ISSUE THREAD

**Issue by dfalster** [NICTA/stateline#48]
Hi, I tried to get stateline (29e0a2ef966139a2ab49f49ba99f8f3bb1d813f6) installed on OSX. 

First problem is with installation of glog (called from `buildPrereqs.sh` , which is called from `fetch-all.sh`). Unfortunately installation of glog-0.3.3 fails on osx, this is a known issue, e.g. https://code.google.com/p/google-glog/issues/detail?id=121, https://github.com/dmlc/ps-lite/issues/7

So i updated to glog-0.3.4 [avaialble on github here](https://github.com/google/glog/archive/v0.3.4.tar.gz) - see  [this commit](https://github.com/traitecoevo/stateline/commit/c4acd0630222a9e47c23cc0ac7900a17edcf88f8) for example code making necessary change. This allows for `buildPrereqs.sh` to complete, but the  `fetch-all.sh` script then fails on [L16](https://github.com/NICTA/stateline/blob/29e0a2ef966139a2ab49f49ba99f8f3bb1d813f6/fetch-all.sh#L16) when trying to build debug and release versions. Error as follows.

```
*** Source directory detected as: 
*** Build directory detected as: /Users/dfalster/Dropbox/_research/SIEF/stateline/build/debug
*** Prerequites directory detected as: /build/prereqs
-- Module path /CMakeModules/
-- Build Type: Unset, assuming Debug
CMake Error at CMakeLists.txt:62 (FIND_PACKAGE):
  By not providing "FindGlog.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Glog", but
  CMake did not find one.

  Could not find a package configuration file provided by "Glog" with any of
  the following names:

    GlogConfig.cmake
    glog-config.cmake

  Add the installation prefix of "Glog" to CMAKE_PREFIX_PATH or set
  "Glog_DIR" to a directory containing one of the above files.  If "Glog"
  provides a separate development package or SDK, be sure it has been
  installed.

-- Configuring incomplete, errors occurred!
```

**Comment by sjhardy** [comment]
Small fix needed still:

```
diff --git a/tools/fetch-deps b/tools/fetch-deps
index 63d516f..fa2f897 100755
--- a/tools/fetch-deps
+++ b/tools/fetch-deps
@@ -21,6 +21,8 @@ if [ $# -eq 0 ]; then
   cd build/prereqs
 fi
 
+PREREQ_DIR=$STATELINE_SOURCE_DIR/build/prereqs
+
 # Create folders for the dependencies
 mkdir -p src
 mkdir -p include
```

**Comment by ltiao** [comment]
Fixed in 9cd9de6fbe

**Comment by dfalster** [comment]
Confirming that now installs on OSX
